api/src/main/groovy/ncei/onestop/api/service/MetadataParser.groovy

```diff
@@ -261,47 +261,43 @@ class MetadataParser {
   }
 
   static Map parseSpatialInfo(GPathResult metadata) {
-    def spatialBounding = [:]
-    def isGlobal = null
     def space = metadata.identificationInfo.MD_DataIdentification.extent.EX_Extent.'**'.find { e ->
       e.@id.text() == 'boundingExtent'
     }.geographicElement
     def bbox = space.'**'.find { it -> it.name() == 'EX_GeographicBoundingBox' }
-    if (bbox) {
 
-      spatialBounding.put('type', 'envelope')
-      spatialBounding.put('coordinates', parseCoords(bbox))
-              //)
-      isGlobal = checkIsGlobal(bbox)
-    }
-    return ["spatialBounding":spatialBounding, "isGlobal" : isGlobal]
+    def spatialBounding = parseBounding(bbox)
+    def isGlobal = checkIsGlobal(spatialBounding)
+
+    return ["spatialBounding": spatialBounding, "isGlobal": isGlobal]
   }
 
-  static def parseCoords(def bbox){
-    def westBound = (bbox.westBoundLongitude == "null" ||  bbox.westBoundLongitude == "") ? null : bbox.westBoundLongitude.Decimal.toFloat()
-    def eastBound = (bbox.eastBoundLongitude == "null" ||  bbox.eastBoundLongitude == "") ? null : bbox.eastBoundLongitude.Decimal.toFloat()
-    def northBound = (bbox.northBoundLatitude == "null" || bbox.northBoundLatitude == "")  ? null : bbox.northBoundLatitude.Decimal.toFloat()
-    def southBound = (bbox.southBoundLatitude == "null" || bbox.southBoundLatitude == "") ? null : bbox.southBoundLatitude.Decimal.toFloat()
+  static def parseBounding(def bbox) {
+    if (!bbox) { return null }
 
-    return [
-          [westBound, northBound],
-          [eastBound, southBound]
-    ]
+    def west = (bbox.westBoundLongitude == "null" ||  bbox.westBoundLongitude == "") ? null : bbox.westBoundLongitude.Decimal.toFloat()
+    def east = (bbox.eastBoundLongitude == "null" ||  bbox.eastBoundLongitude == "") ? null : bbox.eastBoundLongitude.Decimal.toFloat()
+    def north = (bbox.northBoundLatitude == "null" || bbox.northBoundLatitude == "")  ? null : bbox.northBoundLatitude.Decimal.toFloat()
+    def south = (bbox.southBoundLatitude == "null" || bbox.southBoundLatitude == "") ? null : bbox.southBoundLatitude.Decimal.toFloat()
+
+    if (!west || !east || !north || !south) { return null }
```

Comment:
Ditto above comment with this return null

Reply:
See above